concierge.yaml

```diff
@@ -0,0 +1,17 @@
+juju:
+  model-defaults:
+    logging-config: <root>=INFO; unit=DEBUG
```

Comment:
This feels like the only thing not covered by the `microk8s` preset already, perhaps I could just make this the default - seems sensible for testing - and then you could drop this file completely and just run `concierge -p microk8s`